r-package/grf/R/input_utilities.R

```diff
@@ -11,6 +11,36 @@ validate_X <- function(X) {
   if (inherits(X, "Matrix") & !(inherits(X, "dgCMatrix"))) {
       stop("Currently only sparse data of class 'dgCMatrix' is supported.")
   }
+  
+  if (!all(!is.na(X))){
```

Comment:
Small comment, but I find `if (any(is.na(X)) { ... }` clearer (avoids a double negative).

Reply:
agreed!  This will be addressed.